src/main/java/com/bymarcin/openglasses/surface/widgets/component/face/Text.java

```diff
@@ -56,8 +56,8 @@ public IRenderableWidget getRenderable() {
 		@Override
 		public void render(EntityPlayer player, double playerX, double playerY, double playerZ) {
 			GL11.glPushMatrix();
+			GL11.glTranslatef(x, y, 100);
```

Comment:
Isn't 100, like, a lot for Z-order enforcement? Also maybe define a constant instead of just raw number?

Reply:
Items do +50 raw number on z level each time 😛  Does it even matter actually?

Reply:
It doesn't, except for others who set Z level